docs/hadoop-catalog.md

```diff
@@ -10,10 +10,8 @@ license: "This software is licensed under the Apache License version 2."
 
 Hadoop catalog is a fileset catalog that using Hadoop Compatible File System (HCFS) to manage
 the storage location of the fileset. Currently, it supports local filesystem and HDFS. For
-object storage like S3, GCS, and Azure Blob Storage, you can put the hadoop object store jar like
-hadoop-aws into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory to enable the support.
-Gravitino itself hasn't yet tested the object storage support, so if you have any issue,
-please create an [issue](https://github.com/apache/gravitino/issues).
+object storage like S3, GCS, Azure Blob Storage and OSS, you can put the hadoop object store jar like
+`gravitino-aws-hadoop-bundle-{gravitino-version}.jar` into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory to enable the support.
```

Comment:
Why do we need to use the hadoop bundle jar? I assume we already have the Hadoop dependencies in the Hadoop catalog, right?

Reply:
hadoop bundle does not contain `hadoop-aws` though Gravitino severs already provides a Hadoop environment.

Reply:
aws-hadoop-bundle = aws-bundle + hadoop-common + hadoop-aws.